test/test_suite.py

```diff
@@ -1194,19 +1198,46 @@ def test_group_by_composed(self):
         pass
 
 
-class CompoundSelectTest(_CompoundSelectTest):
```

Comment:
To confirm, the `CompoundSelectTest` overrides are being removed because the DBAPI fix has been applied.

I would prefer a separate PR to remove these overrides because I'm 90% sure there are other tests with this skip reason.